pkg/reloader/reloader.go

```diff
@@ -363,6 +416,7 @@ func (r *Reloader) apply(ctx context.Context) error {
 		if r.watchInterval == 0 {
 			return nil
 		}
+
 		r.reloads.Inc()
 		if err := r.triggerReload(ctx); err != nil {
 			r.reloadErrors.Inc()
```

Comment:
Maybe we can add labels to the reload/reloadErrors metrics to specify nature of reload?

Reply:
the reload method should be fixed over time but we could introduce a `reload_info{method="http|signal"} 1` metric?